gradle.properties

```diff
@@ -5,6 +5,6 @@ quarkusPlatformArtifactId=quarkus-universe-bom
 quarkusPluginId=io.quarkus
 quarkusPlatformGroupId=io.quarkus
 quarkusPlatformVersion=2.9.0.Final
-org.gradle.jvmargs=-Xmx2024m -XX:MaxPermSize=512m
+org.gradle.jvmargs=-Xmx2024m -XX:MaxMetaspaceSize=512m
```

Comment:
Could these jvmargs be removed all together?

Reply:
If removed jvmargs will revert to default settings of Xmx512m "-XX:MaxMetaspaceSize=256m"